package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sealink-ecom-engine-css",
-  "version": "1.1.3",
+  "version": "1.1.8",
```

Comment:
I'm happy with this, but in future it would be good to only bump by one version.
But I understand you had a bit of issue due to dev versions.

If you continue to have these dev versions perhaps consider local dev versions like, 1.1.9.dev1, 1.1.9.dev2, etc.

Reply:
YEP!!

Reply:
@mnoack I tried to use the dev version like, 1.1.9.dev1 but it shows error. --> "Invalid version: "1.1.9.dev1"

Reply:
I solved all issues! Ignore my message!